packages/admin/cms-admin/src/blocks/image/EditImageDialog.tsx

```diff
@@ -168,6 +178,28 @@ export function EditImageDialog({ image, initialValues, onSubmit, onClose, inher
                                                 </Field>
                                             </FormSection>
                                         </Box>
+
+                                        {dependencyMap["DamFile"] && damFileId && (
+                                            <Box padding={7} paddingTop={0}>
+                                                <Button
+                                                    variant="text"
+                                                    color="inherit"
+                                                    onClick={async () => {
+                                                        // id is checked three lines above
+                                                        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
```suggestion
```

There is no non-null-assertion here and the check isn't three lines above, but six. Please don't copy/paste code without thinking 😁